content/en/docs/reference/glossary/kube-apiserver.md

```diff
@@ -1,19 +1,23 @@
 ---
-title: kube-apiserver
+title: API server
 id: kube-apiserver
 date: 2018-04-12
 full_link: /docs/reference/generated/kube-apiserver/
 short_description: >
-  Component on the master that exposes the Kubernetes API. It is the front-end for the Kubernetes control plane. 
+  Control plane component that serves the Kubernetes API.
 
-aka: 
+aka:
+- kube-apiserver
 tags:
 - architecture
 - fundamental
 ---
- Component on the master that exposes the Kubernetes API. It is the front-end for the Kubernetes control plane. 
+ The API server is a component of the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} that exposes the Kubernetes API.
+The API server is the front-end for the Kubernetes control plane.
```

Comment:
No hyphen for front end.
If it were an adjective, we would use a hyphen or the single word frontend.

Reply:
Ah yeah (was wrong in the original too). Will fix.

Reply:
(fixed)